{"name": "test_firstTest", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <x.test_myFirstTest.TestMush object at 0x105f4a660>\n\n    def test_firstTest(self):\n        print('First test1')\n        print('Second test')\n>       assert False\nE       assert False\n\nsrc/x/test_myFirstTest.py:5: AssertionError"}, "start": 1745222944581, "stop": 1745222944581, "uuid": "ccfee51c-c613-4cf3-b0a9-c8c0cabda8c8", "historyId": "839d3cdd7006d87de67784eac7b33639", "testCaseId": "839d3cdd7006d87de67784eac7b33639", "fullName": "src.x.test_myFirstTest.TestMush#test_firstTest", "labels": [{"name": "parentSuite", "value": "src.x"}, {"name": "suite", "value": "test_myFirstTest"}, {"name": "subSuite", "value": "TestMush"}, {"name": "host", "value": "Musheer-Khan.local"}, {"name": "thread", "value": "38114-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.x.test_myFirstTest"}]}